### DISCUSSION TITLE
Feature request for datepicker - date range pickers

### DISCUSSION THREAD

**Discussion by dsmailys** [iTwin/iTwinUI#1345] [Feedback]
We'd like to be able to select a month only (without a day and time). How soon could this be done?

**Comment by mayank99** [comment]
If you only need to select a month, then DatePicker is the wrong component. You should instead use a [Select](https://itwin.github.io/iTwinUI/react/?path=/story/input-select--basic) or [ComboBox](https://itwin.github.io/iTwinUI/react/?path=/story/core-combobox--basic).

**Reply by dsmailys** [reply]
Hi, sorry if I misstated. We not only need a month, but a year also. The consumer/user would select a month during a particular year.
Let me try to describe the workflow that we want to achieve.

We have a reporting page. The report makes sense only in a scope of a month or more, meaning that you could generate a report only for a particular month (in the past or current month) or for multiple months.
We currently have a date range selector created (from - to), but it includes the day. We'd like to have something standard for picking a month only.

**Reply by mayank99** [reply]
You can use two `ComboBox`s - one for month and one for year.
